src/model.js

```diff
@@ -0,0 +1,242 @@
+const yelp = require("yelp-fusion");
+const proj4 = require("proj4");
+const _ = require("lodash");
+const async = require("async");
+
+const {
+  koopProviderYelp: { api_key },
+} = require("config");
+const client = yelp.client(api_key);
+const YELP_API_MAX_RADIUS = 40000;
+
+const proj = proj4("GOOGLE", "WGS84");
+
+function Model() {}
+
+// each model should have a getData() function to fetch the geo data
+// and format it into a geojson
+Model.prototype.getData = function (req, callback) {
+  if (req.query && req.query.returnCountOnly) {
+    return callback(null, {
+      type: "FeatureCollection",
+      count: 2001,
+      features: [],
+    });
```

Comment:
I don't understand why the count value is hard-coded and applied and returned here.  Are you certain that any type of request will always return this count?  It seems unlikely - what about the case where a `where` filter is passed that leads to no features returned?  e.g `?where=1=2&returnCountOnly` - that should return a count of `0`.

The FeatureServer dependency handles `returnCountOnly` and packages the response accordingly. Is there a reason not to let it do this job?